### DISCUSSION TITLE
Awesome, Welcome to Livewire Discussions!

### DISCUSSION THREAD

**Discussion by lancepioch** [livewire/livewire#968] [4. General]
Introduce yourself and let us know what cool things you're doing or planning with Livewire 😁

I'm Lance and I help Caleb with the Livewire repos. Aside from my work and this, I help with the [Pterodactyl](https://pterodactyl.io) project. I'm working on a media collection organizer and also a personal finance manager with Livewire functionality that'll both be open sourced in time. It definitely makes developing much more of a breeze in many places.

Catch me on the [Discord](https://discord.gg/FAGgt3)!

**Comment by calebporzio** [comment]
Hey Everyone! 👋

**Comment by nuernbergerA** [comment]
Sup!

**Comment by danharrin** [comment]
Hi everyone! 🤟

**Comment by roni-estein** [comment]
Yo🧑‍💻

**Comment by intrepidws** [comment]
Hey everyone! I'm Craig and lately I've been using Livewire and Alpine to make a card game app (you may have seen Caleb tweet about it a few days ago). My friends and I are running a tournament on the app tonight and, assuming things go well, I'll be releasing it live shortly after.

**Reply by lancepioch** [reply]
It looked pretty slick!
